### PR TITLE
wg-osdk: add backlog grooming and issue triage meeting info

### PR DESCRIPTION
**Pull request by estroz**
/cc @jberkhahn @kensipe @asmacdo

### CODE REVIEW

**Comment by estroz** [comment]
@joelanford added the issue filter as the url for triage meetings

**Review by camilamacedo86 — APPROVED** [review]
/lgtm
/approve

File: sig-list.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ When the need arises, a [new SIG can be created](sig-wg-lifecycle.md)
...
+|[OSDK](wg-osdk/README.md)||* [Eric Stroczynski](https://github.com/estroz), Red Hat<br>* [Jonathan Berkhahn](https://github.com/jberkhahn), IBM<br>* [Ken Sipe](https://github.com/kensipe), KUDO<br>|* [Slack](https://kubernetes.slack.com/messages/operator-sdk-dev)<br>* [Mailing List](https://groups.google.com/forum/#!forum/operator-framework)|* Backlog Grooming Meeting: [Wednesdays at 11:00 PT (Western Time) (biweekly)](https://docs.google.com/document/d/1ujWb-rSJ4JWeHLVxK0WS5ZuSJgeESG42MDeYjSl9Q6U/edit)<br>* Issue Traige Meeting: [Mondays at 11:00 PT (Western Time) (weekly)](https://github.com/operator-framework/operator-sdk/issues?q=is%3Aopen+is%3Aissue+no%3Amilestone+sort%3Acreated-asc)<br>* Regular WG Meeting: [Wednesdays at 09:00 PT (Western Time) (monthly)](https://docs.google.com/document/d/1ujWb-rSJ4JWeHLVxK0WS5ZuSJgeESG42MDeYjSl9Q6U/edit)<br>
```
Comment (jberkhahn) [inline sig-list.md:32]: shouldn't this be 'triweekly' not 'biweekly'?

File: sigs.yaml [hunk 2]
```diff
@@ -75,6 +75,18 @@ workinggroups:
       name: Ken Sipe
       company: KUDO
   meetings:
+  - description: Backlog Grooming Meeting
+    day: Wednesday
+    time: "11:00"
+    tz: PT (Western Time)
+    frequency: biweekly
```
Comment (jberkhahn) [inline sigs.yaml:82]: same here

File: wg-osdk/README.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
 To organize and steer the development of the [Operator SDK](https://github.com/operator-framework/operator-sdk) and related projects.
 
 ## Meetings
+* Backlog Grooming Meeting: [Wednesdays at 11:00 PT (Western Time)](https://docs.google.com/document/d/1ujWb-rSJ4JWeHLVxK0WS5ZuSJgeESG42MDeYjSl9Q6U/edit) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=PT%20%28Western%20Time%29).
```
Comment (jberkhahn) [inline wg-osdk/README.md:14]: same here...?

**Review by kensipe — APPROVED** [review]
/lgtm

**Pull request merged**